### PR TITLE
[RU] Added 'advanced/concurrency' lesson translation.

### PR DESCRIPTION
**Pull request by nscyclone**

### CODE REVIEW

**Comment by doomspork** [comment]
Thank you so much @nscyclone!

@brain-geek or @tetiana12345678, would you mind look over this one too?

File: ru/lessons/advanced/concurrency.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
+---
+layout: page
+title: Параллелизм
+category: advanced
+order: 4
+lang: ru
+---
+
+Одна из сильных сторон Elixir &mdash; поддержка параллелизма. Благодаря ErlangVM параллелизм в Elixir легче, чем вы думали.  В основе модели параллелизма лежат Акторы &mdash; процессы, взаимодействующие с другими процессами путём передачи сообщений.
```
Comment (brain-geek) [inline ru/lessons/advanced/concurrency.md]: "Акторы" is just a CS term, there's no need to capitalize it.

File: ru/lessons/advanced/concurrency.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+```elixir
+iex> spawn(Example, :add, [2, 3])
+5
+#PID<0.80.0>
+```
+
+### Передача сообщений
+
+Для взаимодействия между собой процессы используют сообщения. Для этого существует два компонента: `send/2` и `receive`.  Функция `send/2` позволяет отправлять сообщения PID'y.  Для прослушивания и проверки сообщений используется `receive`.  Если при проверке совпадение не будет найдено, выполнение продолжится беспрерывно.
```
Comment (brain-geek) [inline ru/lessons/advanced/concurrency.md]: "существует два компонента" most likely "две части" were meant - due to the fact that it's function names.

Comment (brain-geek) [inline ru/lessons/advanced/concurrency.md]: прослушивания -> получения

Comment (brain-geek) [inline ru/lessons/advanced/concurrency.md]: "беспрерывно" cann be deleted here and it will make even more sense.

**Comment by brain-geek** [comment]
@nscyclone I've added some notes here.

**Comment by nscyclone** [comment]
Thanks for the review, @brain-geek ! I've fixed mentioned mistakes.

**Pull request merged**